### PR TITLE
Give users list suggestion instead of history suggestions when autoco…

### PR DESCRIPTION
**Pull request by peregrineshahin**
…mplete in events

and explicitly set which fields to fill in for the username and password autocomplete.

### CODE REVIEW

**Comment by ppigazzini** [comment]
DEV updated, ready for tests by users.

**Comment by vdbergh** [comment]
This works well for me on iPad. Thanks! I’ll close my PR.

File: server/fishtest/templates/actions.mak [hunk 1]
```diff
@@ -28,13 +28,28 @@
...
-    <input id="user" type="text" name="user" class="form-control" placeholder="username" value="${request.GET.get('user') if request.GET.get('user') != None else ''}">
+    <input
+      id="user"
+      class="form-control"
+      autocomplete="off"
+      placeholder="Search by username"
+      type="text"
+      name="user"
+      list="users-list"
+      value="${request.GET.get('user') if request.GET.get('user') != None else ''}"
```
Comment (ppigazzini) [inline server/fishtest/templates/actions.mak:42]: Better to use `is not None` here:
```
value="${request.GET.get('user') if request.GET.get('user') is not None else ''}"
```

**Review by ppigazzini — APPROVED** [review]

**Comment by ppigazzini** [comment]
PROD updated, thank you @peregrineshahin :)

**Pull request merged**